pkgs/applications/networking/mailreaders/evolution/evolution/default.nix

```diff
@@ -0,0 +1,134 @@
+{ stdenv
```

Comment:
This should be moved, not duplicated.

Reply:
Yes... I am rebasing to master and cleaning up... sorry for my git mess

Reply:
Fixed now. Moved from the gnome umbrella in the evolution folder